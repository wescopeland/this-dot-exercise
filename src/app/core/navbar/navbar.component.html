<mat-toolbar color="primary" class="tde-navbar mat-elevation-z4">
  <span>This.Exercise</span>
</mat-toolbar>