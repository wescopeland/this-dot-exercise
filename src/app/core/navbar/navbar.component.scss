.tde-navbar {
  background-color: #e84d4a;
  color: white;
}
