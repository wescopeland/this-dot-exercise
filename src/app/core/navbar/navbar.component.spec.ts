import { Spectator } from '@netbasal/spectator';
import { createTestComponentFactory } from '@netbasal/spectator/jest';
import { MatToolbarModule } from '@angular/material/toolbar';

import { NavbarComponent } from './navbar.component';

describe('Component: NavbarComponent', () => {
  let spectator: Spectator<NavbarComponent>;
  const createComponent = createTestComponentFactory<NavbarComponent>({
    component: NavbarComponent,
    detectChanges: false,
    declarations: [],
    imports: [MatToolbarModule],
    componentProviders: []
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('exists', () => {
    expect(spectator.component).toBeDefined();
  });

  it('should render the navbar', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
