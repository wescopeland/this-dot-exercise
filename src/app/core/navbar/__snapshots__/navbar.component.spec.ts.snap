// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: NavbarComponent should render the navbar 1`] = `
<tde-navbar>
  <mat-toolbar
    class="tde-navbar mat-elevation-z4 mat-toolbar"
    color="primary"
  >
    <span>
      This.Exercise
    </span>
  </mat-toolbar>
</tde-navbar>
`;
